### ISSUE TITLE
Error with Makefile

### ISSUE THREAD

**Issue by myzinsky** [TUK-EIT/report-eit-latex#8]
Running make gives following error:

```
./main.tex:32: Missing = inserted for \ifnum.
<to be read again>
                   -
l.32

./main.tex:32:  ==> Fatal error occurred, no output PDF file produced!
Transcript written on main.log.
make: *** [main.aux] Error 1
```

**Comment by cdeschryver** [comment]
Is this now fixed with the merge?

**Comment by cdeschryver** [comment]
fixed in 962f668e1510a8ccf4751e44f80f71149fb84b7f
